doc/how_to/integrations/FastAPI.md

```diff
@@ -19,8 +19,14 @@ conda install fastapi
 :::
 
 :::{tab-item} `pip`
-```pip
-conda install fastapi
+```bash
+pip install fastapi
```

Comment:
@philippjfr This was probably a copy-paste bug before.

Reply:
Good catch thanks!